Prace_domowe/Praca_domowa2/Grupa4/Mateusz Stączek/notebook.ipynb

```diff
@@ -0,0 +1,789 @@
+{
```

Comment:
<ul><li>Jeśli wszystkie są różne, ale bardzo podobne, to może nie jest to żaden zysk</li><li>Czy dla targetu dyskretnego nie można tego zrobić?</li><li>Ok, chociaż można rozważać&nbsp;macierze rzadkie, prawda?</li></ul><p>Jakieś&nbsp;minusy TE?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/75/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<ul><li>if every value in target column was different, applying target encoding would not make any changes to our dataframe,</li></ul><p>Nieprawda! Liczba wartości w kolumnie targetowej nie ma w tym przypadku znaczenia, bo liczymy średnią dla danej grupy</p>

Reply:
<p>A z racji korzystania z targetu mamy podatność na overfitting</p>